debian/default

```diff
@@ -0,0 +1 @@
+ENABLED=yes
```

Comment:
Why not switch this to ""? In that case you don't the problem of the package being installed on clients running you a server.